apps/web/pages/video/[uid].tsx

```diff
@@ -305,11 +305,12 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
   }
 
   //daily.co calls have a 60 minute exit buffer when a user enters a call when it's not available it will trigger the modals
-  const now = new Date();
-  const exitDate = new Date(now.getTime() - 60 * 60 * 1000);
+  const exitDate = dayjs.utc().subtract(1, "hour");
+  const endTime = dayjs.utc(booking?.endTime);
 
   //find out if the meeting is in the past
-  const isPast = booking?.endTime <= exitDate;
```

Comment:
This comparison might now work properly because booking.endTime is always in UTC but exitDate in browser's local time

Reply:
I don't think this is the problem because during this comparison there is a valueOf cast (and it compares the numeric offset from UTC 197..)

Reply:
exitDate is already in UTC format "2024-01-03T05:49:44.703Z". I thought it's in local time format